src/settings.cpp

```diff
@@ -120,6 +415,268 @@ void read_settings(pugi::xml_node* root)
+      source_latest = get_node_value_bool(node_sp, "overwrite_latest");
+      source_separate = source_latest;
+    }
+  } else {
+    // If no <source_point> tag was present, by default we keep source bank in
+    // statepoint file and write it out at statepoints intervals
+    source_separate = false;
+    // TODO: add defaults
+  }
+
+  // TODO: Check source points are subset
+
+  // Check if the user has specified to not reduce tallies at the end of every
+  // batch
+  if (check_for_node(root, "no_reduce")) {
+    reduce_tallies = get_node_value_bool(root, "no_reduce");
+  }
+
+  // Check if the user has specified to use confidence intervals for
+  // uncertainties rather than standard deviations
+  if (check_for_node(root, "confidence_intervals")) {
+    confidence_intervals = get_node_value_bool(root, "confidence_intervals");
+  }
+
+  // Check for output options
+  if (check_for_node(root, "output")) {
+    // Get pointer to output node
+    pugi::xml_node node_output = root.child("output");
+
+    // Check for summary option
+    if (check_for_node(node_output, "summary")) {
+      output_summary = get_node_value_bool(node_output, "summary");
+    }
+
+    // Check for ASCII tallies output option
+    if (check_for_node(node_output, "tallies")) {
+      output_tallies = get_node_value_bool(node_output, "tallies");
+    }
+
+    // Set output directory if a path has been specified
+    if (check_for_node(node_output, "path")) {
+      path_output = get_node_value(node_output, "path");
+      if (!ends_with(path_output, "/")) {
+        path_output += "/";
+      }
+    }
+  }
+
+  // Check for cmfd run
+  if (check_for_node(root, "run_cmfd")) {
+    cmfd_run = get_node_value_bool(root, "run_cmfd");
+  }
+
+  // Resonance scattering parameters
+  if (check_for_node(root, "resonance_scattering")) {
+    xml_node node_res_scat = root.child("resonance_scattering");
+
+    // See if resonance scattering is enabled
+    if (check_for_node(node_res_scat, "enable")) {
+      res_scat_on = get_node_value_bool(node_res_scat, "enable");
+    } else {
+      res_scat_on = true;
+    }
+
+    // Determine what method is used
+    if (check_for_node(node_res_scat, "method")) {
+      auto temp = get_node_value(node_res_scat, "method", true, true);
+      if (temp == "ares") {
+        res_scat_method = RES_SCAT_ARES;
+      } else if (temp == "dbrc") {
+        res_scat_method = RES_SCAT_DBRC;
+      } else if (temp == "wcm") {
+        res_scat_method = RES_SCAT_WCM;
+      } else {
+        fatal_error("Unrecognized resonance elastic scattering method: "
+          + temp + ".");
+      }
+    }
+
+    // Minimum energy for resonance scattering
+    if (check_for_node(node_res_scat, "energy_min")) {
+      res_scat_energy_min = std::stod(get_node_value(node_res_scat, "energy_min"));
+    }
+    if (res_scat_energy_min < 0.0) {
+      fatal_error("Lower resonance scattering energy bound is negative");
```

Comment:
Do we actually care if the lower energy bound is negative?

Reply:
It wouldn't really cause problems, but nevertheless it doesn't make sense so I'm in favor of just leaving the error message.